website/www/site/content/en/contribute/get-started-contributing.md

```diff
@@ -109,7 +109,10 @@ Questions can be asked on the [#beam channel of the ASF Slack](https://beam.apac
 1. Find or create an issue in the [Beam repo](https://github.com/apache/beam/issues/new/choose).
    Tracking your work in an issue will avoid duplicated or conflicting work, and provide
    a place for notes. Later, your pull request will be linked to the issue as well.
-2. Comment on the issue saying that you would like to work on it.
+2. Comment ".take-issue" on the issue. This will cause the issue to be assigned to you.
+   When you've completed the issue, you can close it by commenting ".close-issue".
+   If you are a committer and would like to assign an issue to a non-committer, they must comment
+   on the issue first; please tag the user asking them to do so or to comment ".take-issue".
```

Comment:
That's a good point - that's a limitation as currently implemented.

I added #21755 to address that limitation, it will ignore comments like `.take-issue`